lang/perl-xml-parser/Makefile

```diff
@@ -38,7 +38,7 @@ define Package/perl-xml-parser
   CATEGORY:=Languages
   TITLE:=Perl XML Parser
   URL:=http://search.cpan.org/dist/XML-Parser/
-  DEPENDS:=perl +libexpat +perlbase-dynaloader +perlbase-essential
+  DEPENDS:=+perl +libexpat +perlbase-dynaloader +perlbase-essential
```

Comment:
What about all the other places that have this?

https://github.com/openwrt/packages/blob/master/lang/perl-net-http/Makefile#L32
https://github.com/openwrt/packages/blob/master/lang/perl-html-tagset/Makefile#L32
https://github.com/openwrt/packages/blob/master/lang/perl-cgi/Makefile#L36

etc.

Reply:
No idea. All I know is, even though grilo has perl-xml-parser in PKG_BUILD_DEPENDS, it does not get built.

Reply:
I think we need a better understanding of how to fix this properly and completely.